cloudbuild.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+
+# https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS"
+# BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+steps:
+- name: gcr.io/cloud-builders/git
+  args: ['fetch', '--unshallow']
+- name: 'gcr.io/cloud-builders/gcloud'
+  id: execute notebooks
+  entrypoint: /bin/sh
+  args:
+  - '-c'
+  - |
```

Comment:
What about creating a proper could builder instead of having the script directly in the cloudbuild.yaml so that we can maintain and extend it easily?

Reply:
Thanks @BenoitDherin , I agree this could be a bit more compartmentalized.

I since made some small steps in this direction: now using a Deep Learning Container as a builder. But probably the logic of which notebooks exactly to run has to be moved to a Python script.

I turned this PR into a Draft for now, and let's discuss on Thu